### PR TITLE
Refactored the code, styling and color for all components

### PR DESCRIPTION
**Pull request by j24m**
Refactored the code, styling and color for all components.

### CODE REVIEW

File: Components/card.css [hunk 1]
```diff
@@ -1,157 +1,231 @@
...
-    top:0.5rem;
-    right:0.8rem;
+/* Simple and Image Cards */
+.card {
+    position: relative;
+    background-color: var(--light-color);
+    border: 1px solid var(--button-floatHover-color);
+    width: 30rem;
+    padding: 1rem;
+    border-radius: 0.5rem;
```
Comment (saurabhchirde) [inline Components/card.css:8]: Avoid using border radius in rem, use px.

File: Components/input.css [hunk 2]
```diff
@@ -1,38 +1,35 @@
...
-    border:0.1rem solid var(--primary-color); 
+
+.input-container input{
     padding:1rem;
-}
-input:focus{
-    outline:none;
-    border:0.3rem solid var(--blue-shade-color2);
+    width:20rem;
+    border-radius: 0.5rem;
```
Comment (saurabhchirde) [inline Components/input.css:19]: Here as well.

File: Components/navigation.css [hunk 3]
```diff
@@ -1,53 +1,80 @@
...
+
+.input-field-icon {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 1rem;
+    font-size: 1.5rem;
+    gap: 1rem;
+    border: solid var(--light-color);
+    border-radius: 0.5rem;
```
Comment (saurabhchirde) [inline Components/navigation.css:56]: Here as well.

**Review by saurabhchirde — APPROVED** [review]
Now everything looks great with proper naming conventions with a clean code. 
Good to merge. 


On all cards, If you could fix and align the button position in the same place, then it would improve UI and look consistent throughout the page.


In this, the remove button and favourite icon are overflowing the card.

File: .vscode/settings.json [hunk 4]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```
Comment (PuneetP16) [inline .vscode/settings.json:3]: add it in git ignore.

File: Components/badge.css [hunk 5]
```diff
@@ -1,41 +1,65 @@
...
+}
+
+.badge-idle {
+    bottom: 0;
+    right: 0.4rem;
+    height: 1rem;
+    width: 1rem;
+    background-color: var(--yellow-shade-color2);
+}
+
```
Comment (m26unkwn) [inline Components/badge.css:34]: Things You Can Do Here.
1. You can create separate `badge` class then you can add like colors to it.
e.g badge active, badge idle

**Review by m26unkwn — APPROVED** [review]
Everything is good to me.

**Review by PuneetP16 — APPROVED** [review]
Looks good to merge.

**Pull request merged**